src/Workspaces/CoreTestUtilities/Formatting/FormattingTestBase.cs

```diff
@@ -92,7 +93,7 @@ protected static void AssertFormat(Workspace workspace, string expected, SyntaxN
         protected static void AssertResult(string expected, SourceText sourceText, IList<TextChange> result)
         {
             var actual = sourceText.WithChanges(result).ToString();
-            Assert.Equal(expected, actual);
+            AssertEx.EqualOrDiff(expected, actual);
```

Comment:
Should we just submit this to xunit itself? Seems like it'd be very generally useful.

Reply:
I found https://github.com/xunit/xunit/issues/1713 which seems similar?